openml/utils.py

```diff
@@ -40,6 +42,53 @@ def extract_xml_tags(xml_tag_name, node, allow_none=True):
         else:
             raise ValueError("Could not find tag '%s' in node '%s'" %
                              (xml_tag_name, str(node)))
+
+
+def _tag_entity(entity_type, entity_id, tag, untag=False):
+    """Abstract function that can be used as a partial for tagging entities
+       on OpenML
+
+        Parameters
+        ----------
+        entity_type : str
+            Name of the entity to tag (e.g., run, flow, data)
+
+        entity_id : int
+            OpenML id of the entity
+
+        tag : str
+            The tag
+
+        untag : bool
+            Set to true if needed to untag, rather than tag
+
+        Returns
+        -------
+        tags : list
+            List of tags that the entity is still tagged with
```

Comment:
Maybe write `List of tags that the entity is (still) tagged with` because the number of tags could also increase, right?

Reply:
Agreed